app/lib/models/nav_items.dart

```diff
@@ -1,14 +1,26 @@
 import 'package:flutter/material.dart';
 
-class NavigationItems {
+/// [NavigationItem] holds all information when building [BottomNavigationBarItem].
```

Comment:
Not sure I would keep these comments, I think it obvious from the code what each is doing?

Reply:
Will remove unnecessary comments. I was just following the documentation and they do it like this ahaha